src/main/java/de/uol/pgdoener/civicsage/GlobalExceptionHandler.java

```diff
@@ -28,6 +29,13 @@
 @ControllerAdvice
 public class GlobalExceptionHandler {
 
+    @ExceptionHandler(SplittingException.class)
+    public ResponseEntity<Object> handleSplittingException(SplittingException ex) {
+        ErrorResponse errorResponse = ErrorResponse.create(ex, HttpStatus.BAD_REQUEST, ex.getMessage());
```

Comment:
I would argue, that `BAD REQUEST` is not the best choice in terms of describing what went wrong. But then again, I fail to find another status code that is a better match for the encountered problem.